specification/ws/src/main/scripts/org/kaazing/specification/ws/opening/response.header.sec.websocket.extensions.reordered/handshake.response.rpt

```diff
@@ -0,0 +1,22 @@
+accept http://localhost:8080/path?query
+accepted
+connected
+
+read method "GET"
+read version "HTTP/1.1"
+read header "Host" "localhost:8080"
+read header "Upgrade" /(?i:websocket)/
+read header "Connection" /(?i:Upgrade)/
+read header "Sec-WebSocket-Key" /(?<key>[a-zA-Z0-9+\/=]{24})/
+read header "Sec-WebSocket-Version" "13"
+read header "Sec-WebSocket-Protocol" "primary, secondary"
```

Comment:
According to [RFC-6455](http://tools.ietf.org/html/rfc6455#page-48) the header is 
Sec-WebSocket-Extensions